docs: deprecation message (#4)
DevExpress/TestCafe-RPC: README.md

# ⚠️This package is now deprecated⚠️

> It will receive no future updates. Please visit https://testcafe.io for instructions about installing and using the most recent TestCafe version.

Reviewer: ```suggestion
It will receive no future updates. Please migrate to the latest version of TestCafe. Learn about the latest version of TestCafe on https://testcafe.io. 
```

TestCafe RPC
======

> RPC library for the [TestCafé](http://testcafe.devexpress.com/). It allows you to create a single TestCafé instance and then interact with it programmatically from any machine in your network.

Author: Yes, you are correct.

BTW, I didn't do all these changes intentionally. Likely, they were made automatically by Git due to end-of-line conversions.

var rpcHostname = 'myhostname',
    rpcPort = 1339;

> //Connect to the existent TestCafé server (if you running both client and server on the same machine

Reviewer: ```suggestion
//Connect to the existent TestCafé server (if your client and server are running on the same machine
```

> //hostname-parameter can be ommited).

Reviewer: ```suggestion
//the hostname parameter can be ommited).
```
